isabelle/CodeExports/CodeExports.thy

```diff
@@ -69,6 +72,15 @@ export_code
   validAndPositive_state
 
   \<comment> \<open> Export examples to be used as oracle specificaiton tests\<close>
+  escrowExample
+  everythingIsAlrightTransactions
+  everythingIsAlrightPayments
+  confirmProblemTransactions
+  confirmProblemPayments
+  dismissClaimTransactions
+  dismissClaimPayments
+  confirmClaimTransactions
+  confirmClaimPayments
   swapExample
   happyPathTransactions
   happyPathPayments
```

Comment:
we should probably add a prefix of the example type (would rename the happyPathXX at least)

Reply:
Also, with the runScenarion helper below, it would be great to create a `Scenario` type alias/record and export bundles of transactionXpayments

Reply:
Ok, will do this with the refactoring of the Swap example.